Sources/SmithyXML/Writer/Writer+LibXML2.swift

```diff
@@ -0,0 +1,115 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#if os(Linux)
+    import LibXML2
+#else
+    import libxml2
+#endif
```

Comment:
This will probably break if we ever add support for Windows in future. Not sure if it matters.